spec/bundler/env_spec.rb

```diff
@@ -139,5 +139,9 @@
         expect(described_class.report).to include("Git         1.2.3 (Apple Git-BS)")
       end
     end
+
+    it "properly parses version of tools when shelling out to them" do
```

Comment:
I would just say ` it "strips ruby version of new line characters"`

Reply:
Absolutely, the description should be better. The thing is the test is not specific to `ruby', it's actually `rbenv` and `rvm` that are passed as parameters to the method in real life, but I used `ruby` since I'm sure it will be in PATH during the spec run. Have a look at the new version!